Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -520,8 +521,22 @@ namespace AZ::SceneAPI::Behaviors
 
         // AssetImportRequest
         Events::ProcessingResult UpdateManifest(Containers::Scene& scene, ManifestAction action, RequestingApplication requester) override;
+        Events::ProcessingResult PrepareForAssetLoading(Containers::Scene& scene, RequestingApplication requester) override;
```

Comment:
Where is this called?

Reply:
this is the event handler when  PrepareForAssetLoading is starting to happen.
Code\Tools\SceneAPI\SceneCore\Events\AssetImportRequest.cpp at line 143 calls it